app/helpers/form_helper.rb

```diff
@@ -442,7 +442,7 @@ def get_attr_error(f, attr)
 
   def form_select_f(f, attr, array, select_options = {}, html_options = {})
     addClass html_options, "form-control"
-    include_blank = select_options.delete(:include_blank)
+    include_blank = select_options[:include_blank]
```

Comment:
@MariaAga This will generate a new `<option ../>` tag in the DOM. Is it something we expect to be there, or should it be managed by `select2`?